File: Software Technologies/Exam/csharp/CatShop/Models/Cat.cs
namespace CatShop.Models
{
	using System.ComponentModel.DataAnnotations;

	public class Cat
    {
		[Key]
		public int Id { get; set; }

		public string Name { get; set; }

		public string Nickname { get; set; }

		public double Price { get; set; }

		public Cat()
		{
		}

		public Cat(string name, string nickname, double price)
		{
			this.Name = name;
			this.Nickname = name;
			this.Price = price;
		}
	}
}

File: Software Technologies/Exam/java/src/main/java/cat/entity/Cat.java
package cat.entity;

import javax.persistence.*;
import javax.validation.constraints.NotNull;

@Entity
@Table(name = "cats")
public class Cat {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    public Integer id;

    @Column
    public String name;

    @Column
    public String nickname;

    @Column
    public Double price;

    public Cat() {
    }

    public Cat(String name, String nickname, Double price) {
        this.name = name;
        this.nickname = nickname;
        this.price = price;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getname() {
        return name;
    }

    public void setname(String name) {
        this.name = name;
    }

    public String getnickname() {
        return nickname;
    }

    public void setnickname(String nickname) {
        this.nickname = nickname;
    }

    public Double getprice() {
        return price;
    }

    public void setprice(Double price) {
        this.price = price;
    }
}
